pages/common/git-maintenance.md

```diff
@@ -0,0 +1,24 @@
+# git-maintenance
+
+> Run tasks to optimize Git repository data.
+> More information: <https://git-scm.com/docs/git-maintenance>.
+
+- Initialize Git config values so any scheduled maintenance will start running on this repository:
```

Comment:
```suggestion
- Register the current repository in the user's list of repositories to daily have maintenance run:
```

Reply:
I took my time to understand your suggestion 🍺

Reply:
Probably could've been better, but I had already spent enough time on it 🙂